R/utils-read.R

```diff
@@ -1087,7 +1109,7 @@ readGapTolerances <- function(reportObject, timezone){
 #' @description Reads and validates the primary TS metadata
 #' @param reportObject The full report JSON object
 readPrimaryTSMetadata <- function(reportObject){
-  requiredFields <- c('period', 'utcOffset', 'timeSeriesType', 'description', 'primary', 'publish')
```

Comment:
didn't seem that we were using the primary field, and since it is no longer in the tsmetadata i just yanked it without re-grabbing from the extended attributes